sdf/1.6/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set (sdfs
   actor.sdf
   air_pressure.sdf
+  air_speed.sdf
```

Comment:
Since this is being implemented in `sdf13`, we should only update the spec for `1.10` (ie., remove changes for 1.6, 1.7, 1.8, 1.9)

```suggestion
```